test/integration/multinode_test.go

```diff
@@ -446,11 +447,32 @@ func validateDeployAppToMultiNode(ctx context.Context, t *testing.T, profile str
 			t.Errorf("Pod %s could not resolve 'kubernetes.default': %v", name, err)
 		}
 	}
+
 	// verify both pods could resolve to a local service.
 	for _, name := range podNames {
 		_, err = Run(t, exec.CommandContext(ctx, Target(), "kubectl", "-p", profile, "--", "exec", name, "--", "nslookup", "kubernetes.default.svc.cluster.local"))
 		if err != nil {
 			t.Errorf("Pod %s could not resolve local service (kubernetes.default.svc.cluster.local): %v", name, err)
 		}
 	}
+
+	// verify both pods could resolve "host.minikube.internal"
+	for _, name := range podNames {
```

Comment:
any reason we added it to the multinode test ? how about we add it as a subset of the functional test ?

Reply:
no special reason other than already having other dns tests there, so just to keep them in one place?

Reply:
+= multinode test would also check if pods on a node that does not host coredns will be able to resolve it